doc/03_configuration.md

```diff
@@ -145,6 +145,22 @@ This is what the `:unit` suite looks like after expansion:
 
 If you don't define any test suites then Kaocha assumes a single `:unit` test suite.
 
+#### Disabling Automatic Classpath Handling
+
+Kaocha will add any entries in `:kaocha/test-paths` to the classpath, if they
+aren't on there already. This is done so this information doesn't need to be
+duplicated between `tests.edn` and your project configuration (`deps.edn`,
+`project.clj`, etc).
+
+However, often people will have these paths in their `deps.edn` anyway (e.g. in
+a `:test` or `:dev` profile), so they are accessible for interactive evaluation
+(REPL, in-buffer eval). In some cases you also do not *want* Kaocha to add
+these, because the way we make sure we have a dynamic classloader so we can add
+entries at runtime can confuse third party tools.
```

Comment:
"the way we make sure we have a dynamic classloader so we can add
entries at runtime" is a little wordy (the phrasing isn't bad in itself, it's just long in this context).

Maybe something like "In some cases you also do not *want* Kaocha to add
these, because the automatic classpath handling can confuse third-party tools. It's specifically the way Kaocha ensures we have a dynamic classloader to add these entries dynamically that confuses them." 

That second sentence is probably not necessary. If it is, I think you need more information so it's clear why it's necessary to the reader.